src/features/emojiMod.ts

```diff
@@ -97,6 +98,17 @@ const reactionHandlers: ReactionHandlers = {
         });
       }
     }
+
+    const privateMessageToSender = [
+      `You've received a warning from the moderators on your message in ${message.channel}`,
+      "\n\n",
+      "Your message: \n",
+      `\`${truncateMessage(message.content)}\``,
+      "\n\n",
+      `Link: https://discord.com/channels/${message.guild?.id}/${message.channel.id}/${message.id}`
+    ].join("");
```

Comment:
Just curious, why not a multiline string? This seems much noisier
```suggestion
    const privateMessageToSender = 
      `You've received a warning from the moderators on your message in ${message.channel}

Your message:
\`${truncateMessage(message.content)}\`

Link: https://discord.com/channels/${message.guild?.id}/${message.channel.id}/${message.id}
```

Reply:
There's no specific reason. To be honest I just didn't think about it.

https://github.com/reactiflux/reactibot/blob/546418eea4b7d99a2cc4b324cd141a703a13142e/src/features/emojiMod.ts#L67

I copied this message and modified the text a bit ignoring the fact that there's some filtering going on in the example.